### ISSUE TITLE
Unable to execute Event Handler

### ISSUE THREAD

**Issue by HandyHat** [bundabrg/GeyserReversion#2]
The following warning shows up in console right after it says `EducationServer` or `BedrockServer` listening on `IP`.

```
`[22:33:17 WARN]: [Geyser-Spigot] Unable to execute Event Handler
org.geysermc.connector.event.handlers.EventHandler$EventHandlerException: Unable to execute Event Handler
        at org.geysermc.connector.event.handlers.MethodEventHandler.execute(MethodEventHandler.java:70) ~[?:?]
        at org.geysermc.connector.event.EventManager.triggerEvent(EventManager.java:74) ~[?:?]
        at org.geysermc.platform.spigot.GeyserSpigotPlugin.onEnable(GeyserSpigotPlugin.java:138) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.16.2.jar:git-Paper-"dccaf59a0"]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:380) ~[patched_1.16.2.jar:git-Paper-"dccaf59a0"]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:483) ~[patched_1.16.2.jar:git-Paper-"dccaf59a0"]
        at org.bukkit.craftbukkit.v1_16_R2.CraftServer.enablePlugin(CraftServer.java:499) ~[patched_1.16.2.jar:git-Paper-"dccaf59a0"]
        at org.bukkit.craftbukkit.v1_16_R2.CraftServer.enablePlugins(CraftServer.java:413) ~[patched_1.16.2.jar:git-Paper-"dccaf59a0"]
        at net.minecraft.server.v1_16_R2.MinecraftServer.loadWorld(MinecraftServer.java:468) ~[patched_1.16.2.jar:git-Paper-"dccaf59a0"]
        at net.minecraft.server.v1_16_R2.DedicatedServer.init(DedicatedServer.java:241) ~[patched_1.16.2.jar:git-Paper-"dccaf59a0"]
        at net.minecraft.server.v1_16_R2.MinecraftServer.w(MinecraftServer.java:939) ~[patched_1.16.2.jar:git-Paper-"dccaf59a0"]
        at net.minecraft.server.v1_16_R2.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.2.jar:git-Paper-"dccaf59a0"]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_261]
Caused by: java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_261]
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_261]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_261]
        at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_261]
        at org.geysermc.connector.event.handlers.MethodEventHandler.execute(MethodEventHandler.java:68) ~[?:?]
        ... 12 more
Caused by: java.lang.NoSuchMethodError: com.nukkitx.network.util.EventLoops.commonGroup()Lio/netty/channel/EventLoopGroup;
        at au.com.grieve.reversion.ReversionServer.<init>(ReversionServer.java:67) ~[?:?]
        at au.com.grieve.reversion.ReversionServer.<init>(ReversionServer.java:63) ~[?:?]
        at au.com.grieve.reversion.editions.education.EducationReversionServer.<init>(EducationReversionServer.java:39) ~[?:?]
        at au.com.grieve.geyser.reversion.editions.education.EducationEdition.createReversionServer(EducationEdition.java:61) ~[?:?]
        at au.com.grieve.geyser.reversion.GeyserReversionPlugin.onGeyserStart(GeyserReversionPlugin.java:153) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_261]
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_261]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_261]
        at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_261]
        at org.geysermc.connector.event.handlers.MethodEventHandler.execute(MethodEventHandler.java:68) ~[?:?]
        ... 12 more`
```

Using:
- Paper 1.16.2-rc6
- Geyser-Spigot-v1.16-b29
- GeyserReversion-1.0.5

The warning means that GeyserReversion does not load, but Geyser itself works fine.
It does not occur on the standalone version (Geyser-v1.16-b29), where GeyserReversion works fine
This can be reproduced when the only plugins are Floodgate and Geyser.

**Comment by bundabrg** [comment]
Thanks, this one is tricky but I'll have a think about a solution.

**Comment by bundabrg** [comment]
This should now be resolved from Geyser-v1.16-b30. Re-open if it still occurs.
